### PR TITLE
fix(storage-browser): Fix empty actions menu showing tiny empty list box

### PR DESCRIPTION
**Pull request by joon-won**
#### Description of changes

- Fix actions menu showing tiny empty box when no action is available in the view
- Added test
#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] `yarn test` passes and tests are updated/added
- [ ] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax
- [ ] _If this change should result in a version bump_, [changeset added](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) (This can be done after creating the PR.) This does not apply to changes made to `docs`, `e2e`, `examples`, or other private packages.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-storage/src/components/StorageBrowser/components/DropdownMenu.tsx [hunk 1]
```diff
@@ -63,7 +63,11 @@ export function DropdownMenu({
       <ButtonElement
         aria-label="Menu Toggle"
         className={`${STORAGE_BROWSER_BLOCK_TO_BE_UPDATED}__menu-toggle`}
-        disabled={isDisabled || !items?.length}
+        disabled={
+          isDisabled ||
+          !items?.length ||
+          items?.filter((obj) => obj.isHidden).length == items?.length
```
Comment (pranavosu) [inline packages/react-storage/src/components/StorageBrowser/components/DropdownMenu.tsx:69]: May be better to check if `actions` are empty [here](https://github.com/aws-amplify/amplify-ui/blob/9385d5831602f6fb31964090324b0e8d9f780bbd/packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/LocationDetailViewProvider.tsx#L71)

Comment (joon-won) [reply packages/react-storage/src/components/StorageBrowser/components/DropdownMenu.tsx:69]: yeah, will move the check logic there

File: packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/LocationDetailViewProvider.tsx [hunk 2]
```diff
@@ -66,11 +66,13 @@ export function LocationDetailViewProvider({
     message: hasError ? message : downloadErrorMessage,
   });
 
+  const isNoActionAvailable = actions.every((obj) => obj.isHidden);
```
Comment (jordanvn) [inline packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/LocationDetailViewProvider.tsx:69]: Nit: could `obj` be a more descriptive name? Maybe just `action`?

**Review by jordanvn — APPROVED** [review]

**Review by pranavosu — APPROVED** [review]

File: packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/LocationDetailViewProvider.tsx (hunk 2 above)
Comment (joon-won) [reply packages/react-storage/src/components/StorageBrowser/views/LocationDetailView/LocationDetailViewProvider.tsx:69]: Yeah makes a sense to me, I'll try to use more descriptive names for variable going forward

**Pull request merged**